### PR TITLE
fix: let nrwl workspace migrate fail

### PR DESCRIPTION
**Pull request by WolfSoko**
ref: #373

### CODE REVIEW

**Comment by WolfSoko** [comment]
I'm missing an entry for `nx migrate --run-migrations` in allowedPostUpdateScripts. Is it missing by accident? 
Seems to work anyway.

**Comment by samuelfernandez** [comment]
> I'm missing an entry for `nx migrate --run-migrations` in allowedPostUpdateScripts. Is it missing by accident? Seems to work anyway.

Thanks @WolfSoko for spotting this and contributing the PR. The post update script is [here](https://github.com/ng-easy/renovate-config/blob/main/renovate.config.js#L22) would you please update it as well? With that it would be ready to merge

File: nrwlWorkspace.json [hunk 1]
```diff
@@ -47,7 +47,7 @@
           "npm ci --ignore-scripts",
           "npm run prepare --if-present",
           "npx --no-install nx migrate {{{depName}}} --from={{{depName}}}@{{{replace '^[^\\d]*' '' currentValue}}} --to={{{depName}}}@{{{replace '^[^\\d]*' '' newValue}}}",
-          "npx --no-install nx migrate --run-migrations=migrations.json || true",
+          "[ -f migrations.json ] && npx --no-install nx migrate --run-migrations=migrations.json",
```
Comment (samuelfernandez) [inline nrwlWorkspace.json:50]: Could you please add this to [`allowedPostUpgradeCommands`](https://github.com/ng-easy/renovate-config/blob/main/renovate.config.js#L15)? Thanks!

Comment (WolfSoko) [reply nrwlWorkspace.json:50]: done, please have a look again

File: renovate.config.js [hunk 2]
```diff
@@ -19,6 +19,7 @@ module.exports = {
     '^npx --no-install ng update (@[a-z0-9-~][a-z0-9-._~]*\\/)?[a-z0-9-~][a-z0-9-._~]* --from=\\d+\\.\\d+\\.\\d+ --to=\\d+\\.\\d+\\.\\d+ --migrate-only --allow-dirty --force$',
     '^npx --no-install ng lint --fix$',
     '^npx --no-install nx migrate (@[a-z0-9-~][a-z0-9-._~]*\\/)?[a-z0-9-~][a-z0-9-._~]* --from=(@[a-z0-9-~][a-z0-9-._~]*\\/)?[a-z0-9-~][a-z0-9-._~]*@\\d+\\.\\d+\\.\\d+ --to=(@[a-z0-9-~][a-z0-9-._~]*\\/)?[a-z0-9-~][a-z0-9-._~]*@\\d+\\.\\d+\\.\\d+$',
+    '^\\[ -f migrations\\.json ] && npx --no-install nx migrate --run-migrations=migrations\\.json$',
```
[the remark this answers is not in the document]
Comment (samuelfernandez) [reply renovate.config.js:22]:
```suggestion
    '^\\[ -f migrations\\.json \\] && npx --no-install nx migrate --run-migrations=migrations\\.json$',
```

**Review by samuelfernandez — CHANGES_REQUESTED** [review]

**Comment by WolfSoko** [comment]
@samuelfernandez I think my proposed solution does not work. Somehow the task always fails. With 

`Command failed: [ -f migrations.json ] && npx --no-install nx migrate --run-migrations=migrations.json`

Seems like it is a problem with the [ -f migrations.json ] part. 

Should we revert this? Or inveset time into  a working solution?

**Pull request merged**